experimental/tfhub/tests/test_tfhub.py

```diff
@@ -40,6 +40,32 @@ def test_ssd_model_type():
     assert len(series[0]) == 100
 
 
+def test_ssd_model_type2():
+    inputs_list = [
+        pd.Series(
+            [
+                Image(image_path),
+                Image(image_path),
+                Image(image_path),
+                Image(image_path),
+                Image(image_path),
+                Image(image_path),
+            ]
+        )
+    ]
+    results_list = apply_model_spec(
+        {
+            "name": "tfssd",
+            "uri": f"tfhub:///tensorflow/ssd_mobilenet_v2/2",
+            "modelType": "ssd",
+        },
+        inputs_list,
+    )
+    assert len(results_list) == 1
+    series = results_list[0]
+    assert len(series[0]) == 100
```

Comment:
we have 6 images, adding assertion for the number 6?

Reply:
I found it's actually an array of one `Series`, and the `Series` has a shape `(6,...)`.
I updated the assertion, notify me if it's not expected.